Microsoft.HealthVault.Fhir.UnitTests/ToHealthVaultTests/ObservationToHealthVaultWeightTests.cs

```diff
@@ -36,6 +36,21 @@ public void WeightToHealthVault_Successful()
         }
 
         [TestMethod]
+        public void HeightToHealthVault_Successful()
```

Comment:
I see previous tests for other data types (blood glucose, etc) in the same class .. mm .. better to change the class name ..